src/components/MainContent/index.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import { TaskModel } from './TaskModel';
+import { Content } from './Content';
+import { MainHeader, NonBulletList, NumberList, Passage, Section, SubHeader } from './Components';
+
+export interface Standard {
+  stdCode: string;
+  stdDesc: string;
+}
+
+export interface DOK {
+  dokCode: string;
+  dokDesc: string;
+  dokShort: string;
+}
+
+export interface Stem {
+  stemDesc: string;
+  shortStem: string;
+}
+
+export interface Task {
+  taskName: string;
+  taskDesc: string;
+  examples: string;
+  stimulus: string;
+}
+
+export interface Target {
+  title: string;
+  shortCode: string;
+  description: string;
+  standards: Standard[];
+  DOK: DOK[];
+  targetType: string;
+  clarification: string;
+  heading: string;
+  evidence: string[];
+  vocab: string;
+  tools: string;
+  stimInfo: string;
+  devNotes: string;
+  complexity: string;
+  dualText: string;
+  accessibility: string;
+  stem: Stem[];
+  taskModels: Task[];
+  rubrics: string[];
+}
+
+export interface MainContentProps {
+  target: Target;
+}
+
+export const RenderListItems = (items: string[]) =>
+  items.map((item, index) => {
+    return <li key={index}>{item}</li>;
+  });
+
+const ShortenStandardCode = (code: string) => {
+  const parts = code.split('.');
+  if (parts.length !== 7) return code;
+
+  return `${parts[4]}-${parts[6]}`;
+};
+
+const ColorStandardCode = (code: string) => {
+  const found = code.match('^.*[0-9]$');
+  if (found) {
+    return (
+      <span>
+        {code}
+        <style jsx>{`
+          color: red;
```

Comment:
Invalid CSS. This should be an easy fix, since your root element is a primitive one.